members/admin/person_admin.py

```diff
@@ -346,13 +347,20 @@ def export_csv(self, request, queryset):
                 + family_email
                 + ";"
                 + person.zipcode
+                + ";"
+                + '"'
+                + person.notes.replace('"', '""')
+                + '"'
                 + "\n"
             )
-            response = HttpResponse(result_string, content_type="text/csv")
+            response = HttpResponse(
+                f'{codecs.BOM_UTF8.decode("utf-8")}{result_string}',
+                content_type="text/csv",
```

Comment:
I https://github.com/CodingPirates/forenings_medlemmer/pull/817/files#diff-5b73a7fa34664628762e03c48338c8ffe9dde3cc0cf7f8e45bf8cef51ea3a522R365, satte vi
```
content_type="text/csv; charset=utf-8",
```
Skal vi også eksplicit sætte `utf-8` for dette PR. Tænker i hvert fald det skal være konsistent